dipy/denoise/pca_noise_estimate.pyx

```diff
@@ -47,12 +48,25 @@ def pca_noise_estimate(data, gtab, patch_radius=1, correct_bias=True,
     smooth : int
         Radius of a Gaussian smoothing filter to apply to the noise estimate
         before returning. Default: 2.
+    image_as_samples : bool
+        Whether to use images as rows (samples) for PCA (algorithm in [1]_) or
+        to use images as columns (features).
```

Comment:
I confirm, just add it as `optional`,

no default value to simplify the maintenance. We always forget to change it in both places in case of change.